src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const stylesObject = {};
+
+  const stylesArray = stylesString
+    .split(';')
+    .filter((style) => style.trim() !== '');
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. The current implementation is correct by using `.split` and `.filter`, which are iteration methods. However, be mindful that `.forEach` is also technically a loop, and the checklist suggests avoiding loops. Since `.forEach` does not return a new array, it is acceptable in this context as an exception.